site/docs/install-ubuntu.md

```diff
@@ -28,24 +28,24 @@ The binary installers are on Bazel's [GitHub releases page](https://github.com/b
 The installer contains the Bazel binary<sup>1</sup>. Some additional libraries must 
 also be installed for Bazel to work.
 
-### 1. Install required packages
+### Step 1. Install required packages
+
+First, install the prerequisites: `zip`, `g++`, `zlipb1g-dev`, `unzip`, and `python`.
 
 ```bash
 sudo apt-get install pkg-config zip g++ zlib1g-dev unzip python
 ```
 
-### 2. Download Bazel
-
-**Note:** In the installer file names listed in this document, replace
-`<version>` with the appropriate Bazel version number.
+### Step 2. Download Bazel
 
-Go to Bazel's [GitHub releases page](https://github.com/bazelbuild/bazel/releases).
+Next, download the Bazel binary installer named `bazel-<version>-installer-linux-x86_64.sh` from the [Bazel releases page on GitHub](https://github.com/bazelbuild/bazel/releases).
 
-Download the binary installer `bazel-<version>-installer-linux-x86_64.sh`.
+**Note:** In the installer file name listed in this document, replace
```

Comment:
It's common practice, for example in manpages, that the user has to replace fields in &lt;angle_brackets&gt;, so I suggest either dropping this sentence or appending it to the last sentence in parentheses, e.g.:

> Next, download the Bazel binary installer named `bazel-<version>-installer-linux-x86_64.sh` from the [Bazel releases page on GitHub](https://github.com/bazelbuild/bazel/releases) (where &lt;version&gt; is a Bazel version, e.g. "0.12.0").

Reply:
Removed the note.